doc/build-osx.md

```diff
@@ -20,17 +20,17 @@ Dependencies
 
 NOTE: Building with Qt4 is still supported, however, could result in a broken UI. Building with Qt5 is recommended.
 
-Build Bitcoin Core
+Build Namecoin Core
 ------------------------
 
-1. Clone the bitcoin source code and cd into `bitcoin`
+1. Clone the namecoin source code and cd into `namecoin`
 
-        git clone https://github.com/bitcoin/bitcoin
-        cd bitcoin
+        git clone https://github.com/namecoin/namecoin-core
+        cd namecoin
```

Comment:
@JeremyRand 
Tested it on Mac OS 10.11.16 and confirm that `namecoin-core` is the right one.